rust/src/checkpoints.rs

```diff
@@ -172,6 +163,11 @@ fn parquet_bytes_from_state(state: &DeltaTableState) -> Result<Vec<u8>, Checkpoi
         })
         .collect();
 
+    println!("CREATING CHECKPOINT");
+    for a in state.files().iter() {
+        println!("{}", a.path);
+    }
+
```

Comment:
@mosyp @xianwill looks like these debugging print should have been removed?

Reply:
yes, u're right, will remove then